### PR TITLE
feat: Update 03-rpiraces-plain.md

### PR DESCRIPTION
**Pull request by rpiraces-plain**

### CODE REVIEW

**Review by marktaylor72 — COMMENTED** [review]
LGTM

**Review by bernat-nosas — APPROVED** [review]

File: _slides/03-rpiraces-plain.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-**Hello world!** RPiraces here! 😎
+**Hello world!** RPiraces here! 🦁
```
Comment (piraces) [inline _slides/03-rpiraces-plain.md]: I think this will be better in two lines...

```suggestion
**Hello world!**
RPiraces here! 🦁
```

**Review by piraces — CHANGES_REQUESTED** [review]

File: _slides/03-rpiraces-plain.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-**Hello world!** RPiraces here! 😎
+**Hello world!**
+RPiraces here! 🦁
 
 ![constructocat](https://octodex.github.com/images/constructocat2.jpg)
```
Comment (piraces) [inline _slides/03-rpiraces-plain.md]:
```suggestion
![constructocat](https://octodex.github.com/images/Sentrytocat_octodex.jpg)
```

**Review by piraces — CHANGES_REQUESTED** [review]
Please review...

**Review by piraces — APPROVED** [review]
LGTM!

**Review by emoralesbe — COMMENTED** [review]
Please review (@emoralesbe )
https://octodex.github.com/images/luchadortocat.png

**Pull request merged**